README.md

```diff
@@ -23,6 +23,12 @@ To get a local copy up and running follow these simple example steps.
 - [Github](https://github.com/Somdotta07)
 - [Twitter](https://github.com/Somdotta07)
 - [LinkedIn](www.linkedin.com/in/somdottasarkar)
+
+👤 **Orçun Uğur**
+
+- GitHub: [@githubhandle](https://github.com/luftedar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/orcunugur)
+
```

Comment:
- [ ] your team has 3 members, please make sure all of you are included in the readme file as well.
- [ ] the link to the MIT license is not working please make sure it is working as presented in [the templated](https://github.com/microverseinc/readme-template) provided by Microverse.
- [ ] please include on how to use the project.